app/src/main/java/org/qosp/notes/ui/editor/EditorFragment.kt

```diff
@@ -1051,7 +1051,8 @@ class EditorFragment : BaseFragment(R.layout.fragment_editor) {
 
     /** Gives the focus to the editor fields if they are empty */
     private fun requestFocusForFields(forceFocus: Boolean = false) = with(binding) {
-        if (editTextTitle.text.isNullOrEmpty()) {
+        // Don't set focus to title if already auto-set to 'Untitled' (probably user doesn't want to set title)
+        if (editTextTitle.text.isNullOrEmpty() && textViewTitlePreview.text.toString() != getString(R.string.indicator_untitled).toString()) {
```

Comment:
Actually, you could remove the `if` clause to check if the title is empty. 
Go straight to check if the content is empty or force flag is true

Reply:
Thanks for the review.

Yes, it does affect new notes.
With the `if` clause removed new notes would open with foucus on content (rather than title as before).
That wouldn't be desired I guess?

My thought was the following:
New notes should open as before, with focus on title.
But if user edited content but left the title blank (which wasn't possible before this fix because of the missing FAB) then user probably left the title blank intentionally (doesn't want to set title at this point) and on re-editing the note the focus should be on content. 
That's how it behaves currently with the `if` clause and my 
`...&& textViewTitlePreview.text.toString() != getString(R.string.indicator_untitled).toString()`
addition.

For illustration, with above addition removed (original if clause) it would behave the following instead:
![FAB_no title2](https://user-images.githubusercontent.com/7658194/230140340-1640d642-041e-4b3f-86eb-079c26dc4e27.gif)